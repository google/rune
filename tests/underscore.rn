println 123_456_789
println "%x" % 0x1234_abcd
