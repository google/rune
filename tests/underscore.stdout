123456789
1234abcd
